contracts/stake-cw20-external-rewards/src/contract.rs

```diff
@@ -1284,28 +1266,24 @@ mod tests {
             .query_wasm_smart(&reward_addr, &QueryMsg::Info {})
             .unwrap();
 
-        assert_eq!(res.reward.rewardRate, Uint128::new(1000));
-        assert_eq!(res.reward.periodFinish, 201000);
-        assert_eq!(res.reward.rewardDuration, 100000);
+        assert_eq!(res.reward.reward_rate, Uint128::new(1000));
+        assert_eq!(res.reward.period_finish, 201000);
+        assert_eq!(res.reward.reward_duration, 100000);
 
         // Add funds in middle of period
         app.borrow_mut().update_block(|b| b.height = 151000);
 
-        let reward_funding = vec![coin(200000000, denom.clone())];
+        let reward_funding = vec![coin(200000000, denom)];
         app.sudo(SudoMsg::Bank({
             BankSudo::Mint {
                 to_address: admin.to_string(),
                 amount: reward_funding.clone(),
             }
-        }));
+        }))
+        .unwrap();
         let _res = app
             .borrow_mut()
-            .execute_contract(
-                admin.clone(),
-                reward_addr.clone(),
-                &fund_msg,
-                &*reward_funding,
-            )
+            .execute_contract(admin, reward_addr.clone(), &fund_msg, &*reward_funding)
```

Comment:
Do we really need the `&*reward_funding`?

Reply:
No we don't. Went ahead and removed it